client/src/main/java/org/apache/rocketmq/client/impl/producer/DefaultMQProducerImpl.java

```diff
@@ -491,27 +492,68 @@ public void send(Message msg,
     public void send(final Message msg, final SendCallback sendCallback, final long timeout)
```

Comment:
I notice that this method is marked as deprecated. Is there any need to modify this method? It may be removed in the future,WDYT